gapic-generator/lib/gapic/schema/wrappers.rb

```diff
@@ -268,6 +268,12 @@ def ruby_package
         parent.ruby_package
       end
 
+      # @return [boolean] True if this service is marked as deprecated, false
+      # otherwise.
+      def is_deprecated?
```

Comment:
The service-level deprecation should also be in unit tests and gold tests (like the method-level one)

Reply:
Done.